packages/hint-doctype/src/hint.ts

```diff
@@ -0,0 +1,125 @@
+/**
```

Comment:
Some comments for the messages for `debug` and `report`:

* Use DOCTYPE instead of doctype, that's how the spec mentions it
* Make sure there's always an ending `.`. You've missed a couple.

Reply:
The `DOCTYPE` comment from above also applies to the `README.md`